internal/http/services/owncloud/ocdav/propfind/propfind.go

```diff
@@ -1147,19 +1148,11 @@ func mdToPropResponse(ctx context.Context, pf *XML, md *provider.ResourceInfo, p
 	appendMetadataProp := func(metadata map[string]string, tagNamespace string, name string, metadataPrefix string, keys []string) {
 		content := strings.Builder{}
 		for _, key := range keys {
-			lowerCaseKey := strings.ToLower(key)
+			kebabCaseKey := strcase.ToKebab(key)
 			if v, ok := metadata[fmt.Sprintf("%s.%s", metadataPrefix, key)]; ok {
-				content.WriteString("<")
-				content.WriteString(tagNamespace)
-				content.WriteString(":")
-				content.WriteString(lowerCaseKey)
-				content.WriteString(">")
+				content.WriteString(fmt.Sprintf("<%s:%s>", tagNamespace, kebabCaseKey))
```

Comment:
I haven't benchmarked it myself, but I assume this is slower than the previous code... it's not an incredible hot path but also not totally cold either

Reply:
meh, thanks for pointing that out... will change it back. I was going for better readability. ;-) But yes, that's less important of course.